ci-operator/step-registry/windows/e2e/operator/test/aws-disconnected/pre/windows-e2e-operator-test-aws-disconnected-pre-chain.yaml

```diff
@@ -0,0 +1,19 @@
+chain:
+  as: windows-e2e-operator-test-aws-disconnected-pre
+  steps:
+    - ref: aws-provision-vpc-disconnected    
+    - chain: aws-provision-bastionhost
+    - ref: ipi-conf
+    - ref: ipi-conf-telemetry
+    - ref: ipi-conf-aws-custom-vpc
+    - ref: ipi-conf-proxy 
+    - ref: ipi-conf-aws
+    - ref: ovn-conf
+    - ref: ovn-conf-hybrid-manifest
+    - chain: ipi-install
+    - ref: aws-provision-tags-for-byo-vpc
+    - ref: enable-qe-catalogsource-disconnected
+    - ref: windows-e2e-operator-test-mirror-images
```

Comment:
Do you this it should be useful to add a sanity check to the install steps to ensure the mirrored images are accessible in the cluster?
I found [this step that might be doing the same](https://steps.ci.openshift.org/reference/set-sample-operator-disconnected#line66)